utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -164,7 +164,7 @@
 "Creative Commons Public Domain": CC-PDDC
 "Creative Commons Zero": CC0-1.0
 "Creative Commons": CC-BY-3.0
-"DBAD": LicenseRef-ort-dbad
+"DBAD": LicenseRef-scancode-dbad
```

Comment:
I agree with the change, but it's a breaking one in case users have rules / classifications that use `LicenseRef-ort-dbad`.

Reply:
I have added the breaking change label to the PR.